events/interaction/features/projectHelper/project-create-modal-handler.js

```diff
@@ -0,0 +1,37 @@
+import { MessageFlags } from "discord.js";
+import { displayActiveProjects } from "../../../../utils/displayProjects.js";
+
+export async function handleProjectCreateModal(interaction) {
+  if (!interaction.isModalSubmit()) return;
+  if (interaction.customId !== "create_project_modal") return;
+  if (!interaction.botContext.user) return;
+  const { projectService, userService } = interaction.services;
+  const name = interaction.fields.getTextInputValue("name").trim();
+  const description = interaction.fields.getTextInputValue("description").trim();
+
+  await interaction.deferReply({ flags: MessageFlags.Ephemeral });
+  try {
+    const user = await userService.getOneDiscordId(interaction.user.id);
```

Comment:
Could save the extra db hit:
```suggestion
  const user = interaction.botContext.user;
  if (!user) return;
  const { projectService } = interaction.services;
  const name = interaction.fields.getTextInputValue("name").trim();
  const description = interaction.fields.getTextInputValue("description").trim();

  await interaction.deferReply({ flags: MessageFlags.Ephemeral });
  try {
```